### PR TITLE
fix more network tests

### PR DESCRIPTION
**Pull request by trichter**
I fixed some more of the network tests. I skipped this one clients/iris doctest which I did not get to pass. The same thing is tested in the regular test suite.

This can be merged as is. Someone else needs to figure out those earthworm stuff. I have no clue about it.

### CODE REVIEW

**Comment by d-chambers** [comment]
Looks like the non-network tests are failing due to new warnings cropping up. Perhaps not a big deal, should be easy to fix.

**Comment by d-chambers** [comment]
Looks like you are addressing the failures in #2978

File: obspy/clients/iris/tests/test_client.py [hunk 1]
```diff
@@ -234,14 +234,15 @@ def test_resp(self):
         t2 = UTCDateTime("2008-001T00:00:00")
         result = client.resp("IU", "ANMO", "00", "BHZ", t1, t2)
         self.assertIn(b'B050F03     Station:     ANMO', result)
+        # Exception: No response data available
```
Comment (d-chambers) [inline obspy/clients/iris/tests/test_client.py:237]: Maybe choose a different response so that we can still test this part of the code?

**Comment by trichter** [comment]
@d-chambers Feel free to finish this one :)

**Comment by trichter** [comment]
I implemented one more test in clients.iris module as suggested. I think this can be merged.

**Review by d-chambers — APPROVED** [review]

**Pull request merged**